frontend/src/containers/Addgroup.js

```diff
@@ -0,0 +1,392 @@
+                          placeholder="charlie@chaps.com"/>
+                      </div>
+                      <div>
+                        <div className="Addgroup-label">Twitterhandle </div>
+                         <Input
+                          name={'twitterHandle'}
+                          value={users4.twitterHandle}
+                          onChange={(value) => appendGroupForm({users4: {name: users4.name, email: users4.email, twitterHandle: value}})}
+                          maxLength={50}
+                          placeholder="@charliechaps"/>
+                      </div>
+                    </div>
+                    <div className="flex flex-row">
+                      <div>
+                        <div className="Addgroup-label">Name</div>
+                        <Input
+                          name={'name'}
+                          value={users5.name}
+                          onChange={(value) => appendGroupForm({users5: {name: value, email: users5.email, twitterHandle: users5.twitterHandle}})}
+                          maxLength={50}
+                          placeholder="Charlie Chaplin"/>
+                      </div>
+                      <div>
+                        <div className="Addgroup-label">Email</div>
+                         <Input
+                          name={'email'}
+                          value={users5.email}
+                          onChange={(value) => appendGroupForm({users5: {name: users5.name, email: value, twitterHandle: users5.twitterHandle}})}
+                          maxLength={50}
+                          placeholder="charlie@chaps.com"/>
+                      </div>
+                      <div>
+                        <div className="Addgroup-label">Twitterhandle </div>
+                         <Input
+                          name={'twitterHandle'}
+                          value={users5.twitterHandle}
+                          onChange={(value) => appendGroupForm({users5: {name: users5.name, email: users5.email, twitterHandle: value}})}
+                          maxLength={50}
+                          placeholder="@charliechaps"/>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div className={`Addgroup-Button`} onClick={this.create.bind(this)}>create!</div>
+            </div>
+        </div>
+        <PublicFooter/>
+      </div>
+    );
+  }
+
+  create() {
+    const { groupForm, validateSchema, createGroup, utmSource, notify } = this.props;
+    const attr = groupForm.attributes;
+    const group = {
+      name: attr.name,
+      slug: attr.slug,
+      mission: attr.mission,
+      longDescription: attr.description,
+      logo: attr.logo || '',
+      website: attr.website,
+      data: {
+        utmSource
+      },
+      tags: attr.tags && attr.tags.split(',').map(Function.prototype.call, String.prototype.trim),
```

Comment:
Never saw `map`, `call` and`trim` used like this, it was intriguing only because I did not understand it at first, what the line does is not obvious. I like it, its clever.

However, this will confuse someone who is not well past novice in JS, I recommend keeping it simple:

`attr.tags.split(',').map(x => x.trim())`